databases.go

```diff
@@ -233,3 +234,65 @@ func (c *Client) DatabaseDisableFeature(ctx context.Context, app, addonID, featu
 
 	return res, nil
 }
+
+type DbmsAttributes struct {
+	PasswordEncryption string `json:"password_encryption"`
+}
+
+type DatabaseUser struct {
+	Name           string          `json:"name"`
+	ReadOnly       bool            `json:"read_only"`
+	Protected      bool            `json:"protected"`
+	Password       string          `json:"password,omitempty"`
+	DbmsAttributes *DbmsAttributes `json:"dbms_attributes,omitempty"`
+}
+
+type DatabaseCreateUserParam struct {
+	DatabaseID           string `json:"database_id"`
+	Name                 string `json:"name"`
+	ReadOnly             bool   `json:"read_only"`
+	Password             string `json:"password,omitempty"`
+	PasswordConfirmation string `json:"password_confirmation,omitempty"`
```

Comment:
**question:** well, i know that the api doc specifies that we wait for a password confirmation, but in fact appsdeck-database does not receives this param. are you sure this field is really used?

Reply:
It is a required param from the DB-API